src/main/java/org/mule/service/http/impl/service/util/RedirectUtils.java

```diff
@@ -51,35 +59,55 @@ public static boolean isRedirected(int statusCode) {
   }
 
   /**
-   * @param requestMethod
    * @param responseStatusCode
-   * @return the original request method if the status code is 301, 307 or 308, or GET if it is 302 or 303.
+   * @return false if the status code is 301, 307 or 308, or true if it is 302 or 303.
    */
-  public static String getMethodForStatusCode(String requestMethod, int responseStatusCode) {
+  public static boolean sendAsGet(int responseStatusCode) {
```

Comment:
use a declarative name for the method mustSendAsGet or something like that.  Should this be public?

Reply:
Changed to private.